CMakeLists.txt

```diff
@@ -328,7 +328,7 @@ set(LOVE_SRC_COMMON
 
 if (APPLE)
 	set(LOVE_SRC_COMMON ${LOVE_SRC_COMMON}
-		src/common/macosx.mm
+		src/common/macos.mm

```

Comment:
This file is still `macosx.mm` in the main branch, it's only renamed to `macos.mm` in `12.0-development`. Did you mean to make the PR based against that branch instead of `main`?

Reply:
Yes, I did. Sorry for the inconvenience.